index.html

```diff
@@ -20,43 +17,65 @@
       <header
         class="navbar-wrapper d-flex align-center justify-around box-shadow-lg"
       >
-        <a class="brand-name text-dec-none headline-lg font-wt-bold p-4" href="#">Gamers Bazaar</a>
+        <a
+          class="brand-name text-dec-none headline-lg font-wt-bold p-4"
+          href="#"
+          >Gamers Bazaar</a
+        >
 
         <div class="search-box d-flex align-center justify-between my-8 w-50">
           <input
             type="text"
-            class="search-input w-100 p-2 m-2 text-sm"
+            class="search-input p-2 m-2 text-sm"
```

Comment:
In future you can make class names more user friendly. padding & margin is clear to me but might not be for everyone looking at your website

Reply:
I will make documentation for utility classes in my component library